src/main/java/SearchEngine.java

```diff
@@ -45,28 +47,25 @@ private void printMenu() {
                 0. Exit""");
     }
 
-    public void find(List<String> list, Map<String, List<Integer>> lines) {
+    private void find(List<String> list, Map<String, List<Integer>> lines) {
```

Comment:
Do you need the first method parameter?

Reply:
I don't need second parameter and LineTokenizer class.
I removed this from my project in new version